packages/patternfly-4/react-core/src/components/withOuia/ouia.ts

```diff
@@ -1,6 +1,6 @@
 export const isOUIAEnvironment = (): boolean => {
   try {
-    return (typeof window !== 'undefined' && window.localStorage && window.localStorage.ouia && window.localStorage.ouia.toLowerCase() === 'true') ||
+    return (typeof window !== 'undefined' && window.localStorage && window.localStorage.getItem("ouia:enabled") && window.localStorage["ouia:enabled"].toLowerCase() === 'true') ||
```

Comment:
Looks ok to me, but believe we need an issue to go along with the PR?

Reply:
Ah sorry, let me create it.

Reply:
@dlabaj #3625

Reply:
@dlabrecq #3625